packages/ui/components/localize/src/number/parseNumber.js

```diff
@@ -16,17 +16,19 @@ import { getDecimalSeparator } from './getDecimalSeparator.js';
  * parseNumber('1,234.56'); // method: heuristic => 1234.56
  *
  * @param {string} value Clean number (only [0-9 ,.]) to be parsed
- * @param {object} options
- * @param {string?} [options.mode] auto|pasted
+ * @param {import('../../types/LocalizeMixinTypes.js').FormatNumberOptions} [options] Locale Options
  * @return {string} unparseable|withLocale|heuristic
  */
-function getParseMode(value, { mode = 'auto' } = {}) {
+function getParseMode(value, options = { mode: 'auto' }) {
```

Comment:
This introduces a break change in a subtle way. For whatever reason, when `{}` is passed as the options, `mode` will be `'auto'` in the previous implementation while it would be `undefined` after this change.

Maybe something like below would be safer, if we go for this route.

```suggestion
function getParseMode(value, { mode = 'auto', locale }) {
```